extensions/walletProviders/tron/src/utils.ts

```diff
@@ -0,0 +1,99 @@
+import { keccak_256 } from "js-sha3"
+import { ec as EC } from 'elliptic'
+
+const hexChar2byte = (c: string) => {
+  let d = 0
+
+  if (c >= 'A' && c <= 'F') d = c.charCodeAt(0) - 'A'.charCodeAt(0) + 10
+  else if (c >= 'a' && c <= 'f') d = c.charCodeAt(0) - 'a'.charCodeAt(0) + 10
+  else if (c >= '0' && c <= '9') d = c.charCodeAt(0) - '0'.charCodeAt(0)
```

Comment:
Maybe you could make those char code calculations into a separate helper function with some good name to make it more readable and reusable.